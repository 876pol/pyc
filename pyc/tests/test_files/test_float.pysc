float gf = 8.0;

void modify() {
    gf -= 20;
}

int main() {
    float f = 4.0 * 12 / 2.4;

    print((string) f + "\n");

    f += 4;

    print((string) f + "\n");

    float gf = (float) "95.0";

    print((string) gf + "\n");

    return 0;
}
