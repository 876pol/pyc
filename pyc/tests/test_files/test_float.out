20.0
24.0
95.0
